src/account_call_builder.js

```diff
@@ -16,16 +16,25 @@ export class AccountCallBuilder extends CallBuilder {
     }
 
     /**
+     * @deprecated use accountID method instead
+     */
+    address(id) {
+        console.warn("AccountCallBuilder#address is deprecated, please use AccountCallBuilder#accountID instead");
+        return this.accountID(id);
+    }
+
+    /**
      * Returns information and links relating to a single account.
      * The balances section in the returned JSON will also list all the trust lines this account has set up.
      *
      * @see [Account Details](https://www.stellar.org/developers/horizon/reference/accounts-single.html)
-     * @param {string} accountId For example: `GDGQVOKHW4VEJRU2TETD6DBRKEO5ERCNF353LW5WBFW3JJWQ2BRQ6KDD`
+     * @param {id} accountId For example: `GDGQVOKHW4VEJRU2TETD6DBRKEO5ERCNF353LW5WBFW3JJWQ2BRQ6KDD`
      * @returns {AccountCallBuilder}
      */
-    address(accountId) {
-        this.filter.push(['accounts', accountId]);
-        return this;
+    accountID(id) {
```

Comment:
I know this is super minor thing but it caught my eye. We're using other form of acronyms in camel case in js sdk, ex. `ledgerId, serverUrl` etc. I don't have any preference but I think it should be consistent.